mbzirc_custom/mbzirc_naive_spinning_radar/models/sensors/mbzirc_naive_spinning_radar/model.sdf

```diff
@@ -28,22 +28,26 @@
       </inertial>
       <sensor name="lidar" type="gpu_ray">
         <pose>0 0 0 1.57079 0 0</pose>
-        <update_rate>30</update_rate>
+        <!-- Make sure lidar can fill every bin. For a 60RPM radar 1 rotation
+        takes 1 second. Since the radar has an angular resolution of 1.44degrees
+        in one second there need to be 250 scans :o-->
+        <update_rate>250</update_rate>
```

Comment:
hmmm this is quite expensive on GPU (lidar render updates) and CPU (processing lidar data). One option is to use a 3D lidar instead of a planar one so we get wider vertical FOV per update. Otherwise we would need reduce update rate and sacrifice accuracy.

Reply:
https://github.com/osrf/mbzirc/pull/166 Is an attempt at this. I misunderstood this comment and thought using a [non-spinning 360 degrees 3D lidar](https://github.com/osrf/mbzirc/pull/161), was what you meant.